api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/request/RequestPath.java

```diff
@@ -124,6 +124,20 @@ public boolean isValid() {
         return isValid;
     }
 
+    public boolean isMethodAllowed(String method) {
+        switch (method.toUpperCase()) {
+            case "POST":
+                return resourceId == null;
+            case "PUT":
+            case "PATCH":
+            case "DELETE":
+                return resourceId != null;
+            case "GET":
+            default:
+        }
+        return true;
```

Comment:
Shouldn't this return false for unknown/unsupported methods?

```
    default:
        return false;
```

Reply:
Yeah, good catch. Thanks.